projects/plugins/jetpack/modules/markdown/easy-markdown.php

```diff
@@ -667,7 +667,7 @@ public function transform( $text, $args = array() ) {
 		 * @param string $text Content to be run through Markdown
 		 * @param array $args Array of Markdown options.
 		 */
-		$text = apply_filters( 'wpcom_markdown_transform_pre', $text, $args );
+		$text = apply_filters( 'wpcom_markdown_transform_pre', $text, $args ) ?? '';
```

Comment:
What do you think about adding that fallback before the filter happens?

Reply:
That wouldn't catch the edge case where the value is filtered and returns `null`.

Reply:
Are there broken filter implementations that do that?

Reply:
We know in the wild that there are quite a few instances that result in `$text` being `null`. We don't know what causes them, or if it's before vs. after the filter, but either could be possible and this catches both.

Reply:
Probably the cause is a code path through the function that doesn't `return`.